web/src/components/starter-kits/StepInfo.tsx

```diff
@@ -0,0 +1,25 @@
+import { Heading } from "@/components/njwds-extended/Heading";
+import { Step } from "@/lib/types/types";
+import { ReactElement } from "react";
+
+export const StepInfo = (props: { step: Step; taskNames: string[] }): ReactElement => {
+  return (
+    <div className="flex-basis-100">
+      <div className="margin-top-05 margin-bottom-2">
+        <span className="border-2px border-secondary-vivid-dark text-secondary-vivid-dark radius-lg padding-x-1 padding-y-05">
+          <b>STEP {props.step.stepNumber}</b>
```

Comment:
[sand] @alexander-littleton Let's use `<strong>` rather than `<b>` to embolden these.

Also @adidner, I'd imagine these would semantically be headings that should appear in the page accessibility heading structure, no?

Reply:
I went with `<b>` due to this being a standalone typography tag which I figured conveys some form of emphasis already for screen readers. No problem switching over to `<strong>` since I could see that being a bit more explicit about conveying "this is basically a heading".